components/org.wso2.carbon.utils/src/main/java/org/wso2/carbon/utils/Constants.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wso2.carbon.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Carbon Constants.
+ *
+ * @since 5.2.0
+ */
+public final class Constants {
+
+    public static final String CARBON_HOME = "carbon.home";
+    public static final String CARBON_HOME_ENV = "CARBON_HOME";
+    public static final String CARBON_CONFIG_YAML = "carbon.yaml";
+    public static final String DEPLOYMENT_CONFIG_YAML = "deployment.yaml";
+
+    /**
+     * maven project properties related constants
+     */
+    public static final String PROJECT_DEFAULTS_PROPERTY_FILE = "project.defaults.properties";
+    public static final String MAVEN_PROJECT_VERSION = "MAVEN_PROJECT_VERSION";
+
```

Comment:
Since this is an existing class and I didn't do any changes hence didn't touch the year